<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMessage</class>
 <widget class="QWidget" name="WidgetMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="userNameLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>拒绝刘亦菲</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userAreaLbl">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="text">
    <string>福建 莆田</string>
   </property>
  </widget>
  <widget class="QLabel" name="userTimeLbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="text">
    <string>2020-02-22 16:52:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="handPortrait">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../image.qrc">
     <normaloff>:/images/img/tx2.jpg</normaloff>:/images/img/tx2.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>612</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="collectBtn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white; 
	border: none;
	color: #808080;
}
QPushButton:hover{
	color: #eb7350;
	}  
QPushButton:pressed{
	color: #eb7350;
	}

</string>
      </property>
      <property name="text">
       <string>收藏</string>
      </property>
      <property name="icon">
       <iconset resource="../image.qrc">
        <normaloff>:/images/img/ICON/collect_normal.jpg</normaloff>:/images/img/ICON/collect_normal.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forwardBtn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white; 
	border: none;
	color: #808080;
}
QPushButton:hover{
	color: #eb7350;
	}  
QPushButton:pressed{
	color: #eb7350;
	}

</string>
      </property>
      <property name="text">
       <string>转发</string>
      </property>
      <property name="icon">
       <iconset resource="../image.qrc">
        <normaloff>:/images/img/ICON/forward_normal.jpg</normaloff>:/images/img/ICON/forward_normal.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="commentBtn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white; 
	border: none;
	color: #808080;
}
QPushButton:hover{
	color: #eb7350;
	}  
QPushButton:pressed{
	color: #eb7350;
	}

</string>
      </property>
      <property name="text">
       <string>评论</string>
      </property>
      <property name="icon">
       <iconset resource="../image.qrc">
        <normaloff>:/images/img/ICON/comment_normal.jpg</normaloff>:/images/img/ICON/comment_normal.jpg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="likeBtn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white; 
	border: none;
	color: #808080;
}
QPushButton:hover{
	color: #eb7350;
	}  
QPushButton:pressed{
	color: #eb7350;
	}

</string>
      </property>
      <property name="text">
       <string>赞</string>
      </property>
      <property name="icon">
       <iconset resource="../image.qrc">
        <normaloff>:/images/img/ICON/like_normal.jpg</normaloff>:/images/img/ICON/like_normal.jpg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="userContentLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>610</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>610</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
